packages/runtime/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@popmenu/use-ssr-computation.runtime",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
Removed "SSRCacheProvider" and "getSSRComputation", so it's not backward compatible